lib/caesar.rb

```diff
@@ -1,6 +1,24 @@
+require "pry"
 class Caesar
-  attr_reader :key
+
 
   def initialize
+
+  end
+
+
+  def caesar_cipher(string, shift)
+    array = Array.new
+    array << string.chars
+
+    array.map do |strings|
```

Comment:
This will properly map the characters to letters, but you aren't saving this newly mapped array. It just goes poof. You should say something like
```
char_codes = array.map do |strings|
   strings.ord
end
```